src/routes/Settings.tsx

```diff
@@ -30,6 +40,21 @@ export const SettingsRoute: React.FC = () => {
     });
   }, []);
 
+  useMemo(() => {
+    (async () => {
+      const response = await apiRequestAuth(
+        `${generateGitHubAPIUrl(Constants.DEFAULT_AUTH_OPTIONS.hostname)}`,
+        'GET',
+        accounts.token,
+      );
+
+      console.info("Token's scopes:", response.headers['x-oauth-scopes']);
```

Comment:
Intended?

Reply:
Yeah, it might be useful for people to know their tokens' scopes.

Reply:
Although, we don't make it that easy to access devtools... I'll remove it